File: Python/vowelsAndConstants.py
def vowelsAndConstant(word):
    vowels = ["a", "e", "i", "o", "u"]
    vowelCount = 0
    constantCount = 0

    for i in range(len(word)):
        if word[i] in vowels:
            vowelCount += 1
        else:
            constantCount += 1

    print(vowelCount)
    print(constantCount)


vowelsAndConstant("sequoia")

File: JavaScript/vowelsAndConstants.js
const vowelsAndConstant = word => {
  const vowels = ["a", "e", "i", "o", "u"];
  let vowelCount = 0;
  let constantCount = 0;

  for (let i = 0; i < word.length; i++) {
    if (vowels.includes(word[i])) {
      vowelCount += word[i].length;
    } else {
      constantCount += word[i].length;
    }
  }
  console.log(vowelCount);
  console.log(constantCount);
};

vowelsAndConstant("sequoia");
